### PR TITLE
Set svg fill and color from path style attr

### PR DESCRIPTION
**Pull request by jonnyparris**

### CODE REVIEW

File: svgbasic.go [hunk 1]
```diff
@@ -189,11 +190,65 @@ func pathParse(pathStr string) (segs []SVGBasicSegmentType, err error) {
...
+
 // SVGBasicType aggregates the information needed to describe a multi-segment
 // basic vector image
 type SVGBasicType struct {
 	Wd, Ht   float64
-	Segments [][]SVGBasicSegmentType
+	Segments []EnhancedSegment
+}
+
+type fillState struct {
```
Comment (danielgek) [inline svgbasic.go:205]: i think this should be uppercase `F`

File: svgbasic.go [hunk 2]
```diff
@@ -189,11 +190,65 @@ func pathParse(pathStr string) (segs []SVGBasicSegmentType, err error) {
...
+		}
+		if key == "fill-rule" {
+			fs.isFilled = true
+			fs.fillStrategy = strings.TrimSpace(parsedRule[1])
+		}
+	}
+	return fs, err
+}
+
+type pathType struct {
```
Comment (danielgek) [inline svgbasic.go:245]: same uppercase here, for what i could understood is the right pattern right @jsaguiar ?

**Review by danielgek — CHANGES_REQUESTED** [review]

File: svgbasic.go (hunk 2 above)
Comment (jonnyparris) [reply svgbasic.go:245]: uppercase just means that something is exported. lowercase is private

Comment (danielgek) [reply svgbasic.go:245]: ohhh ok ok gotta get my golang game up xD

File: svgbasic.go [hunk 3]
```diff
@@ -189,11 +190,65 @@ func pathParse(pathStr string) (segs []SVGBasicSegmentType, err error) {
...
+}
+
+// Parse style tags on path e.g. style="stroke: rgb(0,0,0); stroke-width: 1; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: rgb(0,0,0); fill-rule: nonzero; opacity: 1;"
+func getFillState(style string) (fs fillState, err error) {
+	rules := strings.Split(style, ";")
+	fs.isFilled = false
+	for _, rule := range rules {
+		parsedRule := strings.Split(rule, ":")
+		key := strings.TrimSpace(parsedRule[0])
+		if key == "fill" {
```
Comment (jsaguiar) [inline svgbasic.go]: switch use seems more appropriate here

Comment (jsaguiar) [reply svgbasic.go]: also seems appropriate to put all logic in a different method

**Review by jsaguiar — CHANGES_REQUESTED** [review]

**Review by danielgek — APPROVED** [review]

**Pull request merged**